### PR TITLE
feature: add test instance notification

### PR DESCRIPTION
**Pull request by rekt-hard**
#### Side
always visible due to fixed position not interfering with UI
image

#### Top
sticky until certain point (would have to check further why it's not staying all the way down)
image

### CODE REVIEW

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/header.less [hunk 1]
```diff
@@ -247,3 +247,24 @@ svg:not(:root) {
 span.home-inline{
   white-space: nowrap;
 }
+
+.test-instance-side-sticky{
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/header.less:251]: try using semantic ui - for sticky classes. https://semantic-ui.com/modules/sticky.html

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html [hunk 2]
```diff
@@ -56,6 +56,12 @@
     {%- endblock head %}
   </head>
   <body ng-csp {% if body_css_classes %} class="{{ body_css_classes|join(' ') }}"{% endif %}{% if g.ln %} lang="{{ g.ln.split('_', 1)[0]|safe }}"{% if rtl_direction %} {{ rtl_direction|safe }}{% endif %}{% endif %} itemscope itemtype="http://schema.org/WebPage" data-spy="scroll" data-target=".scrollspy-target">
+    {% if not config.INVENIO_THEME_TUGRAZ_PRODUCTION %}
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html:59]: We would likely be using the env variable to define these configs.
*Test* using ```env``` for defining the variable.
* create a ```.env``` file in your base instance directory , and assign the variable to true : ``` INVENIO_THEME_TUGRAZ_PRODUCTION=true```.
* test if it works.

**Review by mb-wali — CHANGES_REQUESTED** [review]
@utnapischtim  see screenshots which one you prefer - also if the colors or good?

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/header.less (hunk 1 above)
Comment (rekt-hard) [reply invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/header.less]: thank for the hint

**Comment by rekt-hard** [comment]
Looks like this right now

image

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/variables.less [hunk 3]
```diff
@@ -67,3 +67,7 @@
 
 // input
 @outlineColor: #85B7D9;
+
+// notification
+@notificationBackground: #fffaf3;
+@notificationBorder: #e2d5c2;
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/variables.less:73]: add a new line at the end of the file.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html [hunk 4]
```diff
@@ -56,6 +56,12 @@
     {%- endblock head %}
   </head>
   <body ng-csp {% if body_css_classes %} class="{{ body_css_classes|join(' ') }}"{% endif %}{% if g.ln %} lang="{{ g.ln.split('_', 1)[0]|safe }}"{% if rtl_direction %} {{ rtl_direction|safe }}{% endif %}{% endif %} itemscope itemtype="http://schema.org/WebPage" data-spy="scroll" data-target=".scrollspy-target">
+    {% if not config.THEME_TUGRAZ_PRODUCTION %}
+    <div class="ui sticky test-instance">
+      <h4>Test Instance</h4>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html:61]:
```suggestion
      <h4>{{_ {"Test Instance")}}</h4>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html [hunk 5]
```diff
@@ -6,9 +6,14 @@
   details.
 #}
 
+{% if not config.THEME_TUGRAZ_PRODUCTION %}
+<div class="ui sticky test-instance">
+  <h4>Test Instance</h4>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html:11]:
```suggestion
  <h4>{{_ {"Test Instance")}}</h4>
```

**Review by mb-wali — CHANGES_REQUESTED** [review]
That looks nice. I left some comments.

**Comment by rekt-hard** [comment]
closes #187

**Review by mb-wali — APPROVED** [review]

**Pull request merged**